dynamic/main.py

```diff
@@ -73,4 +77,4 @@ def catch_all(path):
 
 
 if __name__ == "__main__":
-    app.run(debug=True, host='0.0.0.0', port=8001)
+    app.run(debug=os.getenv("DEBUG", False), host='0.0.0.0', port=int(os.getenv("PORT", 8001)))
```

Comment:
```suggestion
    app.run(debug='DEBUG' in os.environ, host='0.0.0.0', port=int(os.getenv("PORT", 8001)))
```

This might be personal preference but I prefer to check for the presence of boolean environment variables in case someone does something like `DEBUG=False` which would be truthy in Python since it's a non empty string.

Reply:
To clarify, this wouldn't actually fix the problem of someone doing `DEBUG=False`, but it makes it more explicit that "debug mode is true if `DEBUG` exists" rather than using the _value_ of `DEBUG`.